src/applications/check-in/travel-claim/pages/travel-review/index.jsx

```diff
@@ -50,11 +51,25 @@ const TravelReview = props => {
         <dt className="vads-u-margin-top--2p5">{t('what-youre-claiming')}</dt>
         <dd className="vads-u-margin-top--0p5">
           <span data-testid="claim-info">
-            {`${t('mileage-only-reimbursement-for')} ${
-              appointmentToFile.clinicStopCodeName
-                ? ` ${appointmentToFile.clinicStopCodeName}`
-                : ` ${t('VA-appointment')}`
-            } at ${appointmentToFile.startTime}`}
+            {`${t('mileage-only-reimbursement-for-your')} ${t(
```

Comment:
Same as the other comment above. I'm not sure if this is supposed to say `in person` or not.

Reply:
Correct again, good catches!